waku/waku.md

```diff
@@ -215,17 +213,9 @@ received and validated the Status message. Note that its peer might not be in
 the same state.
 
 When a node is receiving other Waku messages from a peer before a Status
-message is received, the node MUST ignore these messages and SHOULD disconnect
-from that peer.
-Status messages received after the handshake is completed MUST also be ignored.
-
-The fields `bloom-filter`, `light-node`, `confirmations-enabled` and `rate-limits` are OPTIONAL. However if an optional field is specified, all subsequent fields MUST be specified in order to be unambiguous.
-
-The tuple `[ topic-node-enabled topic-interest ]` is also OPTIONAL. If `topics-node-enabled` is set to 1, `topic-interest` setting takes precedence over `bloom-filter`. By default, `topic-node-enabled` MUST be 0.
-
-New Status logic:
+message is received, the node MUST ignore these messages and SHOULD disconnect from that peer. Status messages received after the handshake is completed MUST also be ignored.
 
-Only version and options list is required. All parameters inside option list are specified in an association list and are OPTIONAL. Ordering of key-value pairs is not guaranteed.
+The status message MUST contain an associated list containing various options. All options within this associated list are OPTIONAL, ordering of the key-value pairs is not guaranteed and therefore MUST NOT be relied on.
```

Comment:
I'd still call this a association list as that's the technical term https://en.wikipedia.org/wiki/Association_list but whatever if you prefer associated :p